corehq/apps/app_manager/views/modules.py

```diff
@@ -1352,18 +1353,20 @@ def _init_biometrics_identify_module(app, lang, enroll_form_id):
 
     form_name = _("Followup with Person")
 
+    output_tag = mark_safe(  # nosec: no user-input
+        "<output value=\"instance('casedb')/casedb/case[@case_id = "
+        "instance('commcaresession')/session/data/case_id]/case_name\" "
+        "vellum:value=\"#case/case_name\" />"
+    )
+
     context = {
         'xmlns_uuid': generate_xmlns(),
         'form_name': form_name,
         'lang': lang,
-        'placeholder_label': mark_safe(_(
+        'placeholder_label': format_html(_(
             "This is your follow up form for {}. Delete this label and add "
             "questions for any follow up visits."
-        ).format(
-            "<output value=\"instance('casedb')/casedb/case[@case_id = "
-            "instance('commcaresession')/session/data/case_id]/case_name\" "
-            "vellum:value=\"#case/case_name\" />"
-        ))
+        ), output_tag)
```

Comment:
Eyeballing this it looks like it ends up with the same output as intended, but I'm curious if you checked that somewhere, either in tests or manually, as it's the kind of place I don't think I trust my eyeballs

Reply:
I'll take a closer look. I think I just eyeballed this as well. If it doesn't fit cleanly into a test, I'll make sure it reproduces correctly

Reply:
Verified that it works locally. Testing would have involved either refactoring or setting up a bunch of dependencies